TrkHitReco/fcl/prolog_trigger.fcl

```diff
@@ -65,29 +60,29 @@ TTSmakePH : {
 # combine panel hits in a station
 TTmakeSTH : {
     @table::makeSTH
-    TestFlag : false
+    TestFlag           : false
     ComboHitCollection : "TTSmakePH"
 }
 TTflagBkgHits : {
-    module_type : FlagBkgHits
-    ComboHitCollection : "TTmakePH"
-    StrawHitCollection : "TTmakeSH"
-    BkgMVA : { MVAWeights : "TrkHitReco/data/BkgMVAPanel.weights.xml" }
-    MVANames : [ "HitRho", "HitRhoSpread", "ClusterRho", "ZMin", "ZMax", "ZGap", "NPlanes", "PlaneFraction","NHits" ] 
-    TLTClusterer : { @table::TLTClusterer 
-	TestFlag : false}
-    TNTClusterer : { @table::TNTClusterer
-	TestFlag : false}
-    # use TNTClusterer
-    Clusterer : 2
-    FilterOutput : true
-    FlagComboHits : true
-    FlagStrawHits : false 
-    ComboHitCollection : "TTmakePH"
-    BkgMVA : {
-	MVAWeights : "TrkHitReco/data/BkgMVAPanel.weights.xml"
-    }
+
+  module_type          : FlagBkgHits
+  ComboHitCollection   : "TTmakePH"
+  StrawHitCollection   : "TTmakeSH"
+  ClusterAlgorithm     : 1
+  TNTClustering        : { @table::TNTClusterer }
+  MinActiveHits        : 3
+  MinNPlanes           : 2
+  #MaxIsolated          : 0  
+  ClusterPositionError : 10
+  BkgMVA               : { MVAWeights : "TrkHitReco/data/BkgMVAPanel.weights.xml" }
+  BkgMVACut            : 0.5
+  FilterOutput         : true
+  FlagComboHits        : true
+  FlagStrawHits        : false
+  BackgroundMask       : ["Background"]
+  StereoSelection      : ["Stereo","PanelCombo"]
```

Comment:
I just checked quickly the code and I believe that the "Stereo" bit is not needed anymore because we stopped using the MakeStereoHits module a while ago. @brownd1978 can you please confirm that the Stereo bit can be removed?

Reply:
One question for @gianipez. Does the prolog trigger stop with the KF seed fit? If so, I think we don't need to propagate the flag hits for the strawhits, only the combohits would be needed. This will save some non-negligible time.

Reply:
These flags refer to ComboHits, so after the helix-search, they are not used. My point is that the bit "Stereo" is assigned by the MakeStereoHits module, which is no longer part of the offline, nor the online reconstruction sequences. So, for logical cleanup, I would like to see that removed from the prolog files.
@kutschke does it answer your question?